src/oemof/visio/energy_system_graph.py

```diff
@@ -325,30 +409,92 @@ def connect(self, a, b):
         b: `oemof.solph.network.Node`
             An oemof node (usually a Bus or a Component)
         """
-        if not isinstance(a, Bus):
-            a = fixed_width_text(a.label, char_num=self.txt_width)
+
+        # Back-compatibility with oemof-solph 0.6.0 and lower
+        if SUBNETWORK_MODULE is False:
+            for n in (a, b):
+                if not hasattr(n, "depth"):
+                    setattr(n, "depth", 0)
```

Comment:
Same as above.

Reply:
Removed in f726c9aed629d53a0cc58afd7667b83cf3a9176f